packages/deploy-helpers/deploy-minime.js

```diff
@@ -0,0 +1,16 @@
+const MiniMeTokenFactory = await artifacts.require('@aragon/minime/build/contracts/MiniMeTokenFactory')
```

Comment:
This util is somewhat relevant. I wonder if it should be part of `@aragon/minime` 🤔